### ISSUE TITLE
D x O lopsided

### ISSUE THREAD

**Issue by BronzeMantis** [uaktags/OpenThrone#41]
I was getting a stat boost from buying a weapon in my D Armory without assigning a new unit,

just trained a Soldier and got 54 O
Just trained a Knight and got 3 O
Just trained a Soldier and got 44 O
Just equipped and trained a Soldier and got 84 O
Just equipped 1 set of Armor and trained 1 Soldier and 1 knight and got 88 O
Just equipped and trained a Guard and got 63 D

**Comment by uaktags** [comment]
I'm not sure I'm understanding what you're trying to show. You provided what you did and what you got, but didn't state what you're expecting. You said you got a boost from buying a D weapon, but you didn't make it clear what you bought, just a bunch of Offense guys, possibly an Offense item, and then equipped and trained a defense unit (guard).

**Comment by BronzeMantis** [comment]
The title is a bit misleading as I started typeing differently before I gave it some more testing.

All of the stats are for a full set of weapons, not sure what the exact expectations are until DC is back up

I'd expect a minimum of 25 O for a knight plus their full set of weapons but got 3 O
I'd expect when training a fully equipped Soldier for the stat to be the same each time however I got 44 O and 54 O and 84 O
I had 102 Knights, 102 Soldiers and full set 102 Armory 1 O weapons. When the Armory 1 O was below 102 fully set I got differing stats to what I get now.

Can't see how much I'm getting now because DC is down.
